app/controllers/storage_locations_controller.rb

```diff
@@ -311,6 +314,21 @@ def storage_location_annotation_notification(old_text = nil)
     )
   end
 
+  def set_inline_name_editing
+    location = @parent_location if defined?(@parent_location)
+    location = @storage_location if defined?(@storage_location)
```

Comment:
you don't need to use defined? for instance variables, they are always nil if not set

you can just do 

`location = @storage_location || @parent_location`